project.clj

```diff
@@ -9,4 +9,17 @@
                                   [org.clojure/clojure "1.4.0"]]
                    :eval-in :leiningen}
              :1.2 {:dependencies [[org.clojure/clojure "1.2.1"]]}}
-  :lein-release {:deploy-via :clojars})
+  :lein-release {:deploy-via :clojars}
```

Comment:
Why is it needed?

Reply:
I don't think it's needed. It seems to be [a separate plugin](https://github.com/relaynetwork/lein-release#deploy-via) that the original Leinjacker used.

I was just going to leave it as-is, but it shouldn't hurt to remove it either.